packages/pilot/stories/components/ProductShipping/index.js

```diff
@@ -0,0 +1,99 @@
+import React, { Component } from 'react'
+import { Card } from 'former-kit'
+import { contains, split, when } from 'ramda'
+import { action } from '@storybook/addon-actions'
+
+import Section from '../../Section'
+import ShippingFormComponent from '../../../src/containers/CreateTransaction/Products/Shipping'
+
+const splitNameIfHasDot = when(
+  contains('.'),
+  split('.')
+)
+
+const mergeStateObject = (inputName, { value }) => {
+  const name = splitNameIfHasDot(inputName)
+
+  if (typeof name === 'string') {
```

Comment:
já que tá no ramdinha já dá para trocar isso por um `is` 😄 
```suggestion
  if (is(String, name)) {
```